public/app/plugins/panel/pluginlist/module.html

```diff
@@ -10,14 +10,16 @@ <h6 class="pluginlist-section-header">
           <span class="pluginlist-title">{{plugin.name}}</span>
           <span class="pluginlist-version">v{{plugin.info.version}}</span>
         </span>
-        <span class="pluginlist-message pluginlist-message--update" ng-show="plugin.hasUpdate" ng-click="ctrl.updateAvailable(plugin, $event)" bs-tooltip="'New version: ' + plugin.latestVersion">
-          Update available!
-        </span>
-        <span class="pluginlist-message pluginlist-message--enable" ng-show="!plugin.enabled && !plugin.hasUpdate">
-          Enable now
-        </span>
-        <span class="pluginlist-message pluginlist-message--no-update" ng-show="plugin.enabled && !plugin.hasUpdate">
-          Up to date
+        <span ng-if="ctrl.isAdmin">
```

Comment:
Do not show those action for non admins. Even though they all takes you to plugins page which for non admins shows only readme it looks weird if you have `Enable` button here.